blueocean-material-icons/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@jenkins-cd/blueocean-material-icons",
+  "version": "0.0.2",
+  "description": "Material icons for the Jenkins Design Language",
+  "main": "dist/js/components/index.js",
+  "scripts": {
+    "lint": "gulp lint",
+    "gulp": "gulp build",
+    "mvnclean": "gulp clean",
+    "mvnbuild": "gulp lint build bundle",
+    "mvnbuild:fast": "gulp bundle",
+    "mvntest": "gulp test",
+    "bundle:watch": "gulp bundle:watch",
+    "test": "gulp test",
+    "test:debug": "node --debug-brk ./node_modules/.bin/gulp test:debug",
+    "test:fast": "gulp test:fast"
+  },
+  "author": "Nicolae Pascu <npascu@cloudbees.com> (https://www.cloudbees.com/)",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jenkinsci/blueocean-plugin.git"
+  },
+  "dependencies": {
+    "linkifyjs": "2.1.4",
+    "lodash.debounce": "4.0.8",
```

Comment:
Can we trim down the deps and devDeps here to the minimum that's actually needed?